interface/forms/procedure_order/common.php

```diff
@@ -371,6 +372,18 @@ function getListOptions($list_id, $fieldnames = array('option_id', 'title', 'seq
     }
 
     $alertmsg = '';
+    $isDorn = false;
+    $dornConnector = null;
+    try {
+        $isDorn = DornGenHl7Order::isDornLab($ppid);
+        if ($isDorn) {
+            $dornConnector = new DornGenHl7Order();
+        }
+    } catch (Exception $e) {
+        $isDorn = false;
+        $dornConnector = null;
+    }
+
```

Comment:
@adunsulag This was my work around. Simply disable isDorn flag is class doesn't exist.
Looking at today I started to create events which is basically done however reviewing further I realize this class can stand on its own. Its the same functions we use for other labs like quest/labcorp in interface/procedure_tools.
So moving the classes there still follows our directory structure for procedures plus may be useful and cleaner to use this class for other hl7 generations. Class was nicely done and is clearer of its concern.

Events or Move I'm good either way, choose.

Reply:
I decided events as Sherwin used events